nupic/algorithms/anomaly.py

```diff
@@ -72,7 +72,11 @@ class Anomaly(object):
   _supportedModes = (MODE_PURE, MODE_LIKELIHOOD, MODE_WEIGHTED)
 
 
-  def __init__(self, slidingWindowSize=None, mode=MODE_PURE, binaryAnomalyThreshold=None):
+  def __init__(self, 
+               slidingWindowSize=None,
+               mode=MODE_PURE,
+               binaryAnomalyThreshold=None,
+               claBurnInPeriod=300):
```

Comment:
see https://github.com/numenta/nupic/pull/2267#discussion_r33456034 
I think this makes sense: 
- [ ] propagate setting of `claBurnInPeriod` from `Anomaly` to respective instances of MovingAverage, Likelihood. 
- [ ] on `claBurnIn = 0` `isReady()` would return always True, that way you get 100% prev. behavior.